benchmarks/src/bin/tpch.rs

```diff
@@ -1118,6 +1118,20 @@ mod ci {
                             )
                         }
                     }
+                    (
+                        ScalarValue::Decimal128(Some(l), _, s),
+                        ScalarValue::Decimal128(Some(r), _, _),
+                    ) => {
+                        let tolerance = 0.1;
```

Comment:
tolerance is already defined above in the `float` path -- maybe we could make it a constant or something. Very minor